w2/zadania/MaciejBinczarowski/Zadanie4.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+
+class Zadanie4{
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        switch(scanner.nextInt()){
+            case 1:
+                System.out.println("Zima chyba");
+                break;
+            
+            case 2:
+                System.out.println("Zima chyba");
+                break;
+
+            case 3:
+                System.out.println("Zima chyba");
+                break;
+
+            case 4:
+                System.out.println("Wiosna");
+                break;
+
+            case 5:
+                System.out.println("Wiosna");
+                break;
+
+            case 6:
+                System.out.println("Wiosna");
+                break;
+
+            case 7:
+                System.out.println("Lato");
+                break;
+
+            case 8:
+                System.out.println("Lato");
+                break;
+
+            case 9:
+                System.out.println("Lato");
+                break;
+
+            case 10:
+                System.out.println("Jesień");
+                break;
+
+            case 11:
+                System.out.println("Jesień");
+                break;
+
+            case 12:
+                System.out.println("Jesień");
+                break;
+
+            default:
+                System.out.println("Nie ma takiego miesiąca, no co Ty");
+        }
+
+        // Nie wiem czy poniższy fragment działa, ponieważ mam za starą wersję javy i nie potrafię jej zaktualizować :(
```

Comment:
Po prostu pobierz nowsza wersje i ją zainstaluj, windows sam ogarnie o co chodzi

Reply:
A wiesz może jak na linuxie to zrobić korzystając z wsl i Visual Studio?